.vsts-dotnet.yml

```diff
@@ -78,7 +82,7 @@ stages:
                 -officialBuildId $(Build.BuildNumber)
                 -officialSkipApplyOptimizationData $(SkipApplyOptimizationData)
                 /p:RepositoryName=$(Build.Repository.Name)
-                /p:VisualStudioIbcSourceBranchName=$(IbcSourceBranchName)
+                /p:VisualStudioIbcSourceBranchName=SourceBranch
```

Comment:
Doesn't this need to be accessed via $(SourceBranch)?

Reply:
Yep! Thanks